pandas/_libs/period.pyx

```diff
@@ -10,19 +10,19 @@ from numpy cimport (int8_t, int32_t, int64_t, import_array, ndarray,
                     NPY_INT64, NPY_DATETIME, NPY_TIMEDELTA)
 import numpy as np
 
-cdef extern from "datetime_helper.h":
-    double total_seconds(object)
-
 from libc.stdlib cimport free
 
 from pandas import compat
 from pandas.compat import PY2
+from pandas.core.dtypes.generic import ABCDateOffset
```

Comment:
let's keep perf changes separate. I would much prefer PR's that don't mix multiple changes all at once.